### PR TITLE
fix: update version of eth-ape to >=0.6.5 and <0.7.0

### PR DESCRIPTION
**Pull request by sabotagebeats**
### What I did

update requirements for eth-ape for version 0.6

fixes: #3 
fixes: #APE-711

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [x] New test cases have been added and are passing
- [x] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by sabotagebeats** [comment]
I don't believe this should require new tests or documentation updates?

**Review by fubuloubu — APPROVED** [review]
As long as it looks similar to polygon and BSC plugins we should be good

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,9 @@
     author_email="admin@apeworx.io",
     url="https://github.com/ApeWorX/ape-avalanche",
     include_package_data=True,
-    install_requires=[],
+    install_requires=[
+        "eth-ape>=0.6.0,<0.7.0",
```
Comment (unparalleled-js) [inline setup.py:60]: To encourage using a less buggy version, we could change 0.6.0 to 0.6.5

**Review by unparalleled-js — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (sabotagebeats) [reply setup.py:60]: @unparalleled-js sounds good to me

**Pull request merged**